openpype/hosts/maya/plugins/load/load_reference.py

```diff
@@ -152,3 +152,15 @@ def _post_process_rig(self, name, namespace, context, options):
                 options={"useSelection": True},
                 data={"dependencies": dependency}
             )
+
+
+class AugmentedModelLoader(ReferenceLoader):
+    """Load augmented model via Maya referencing"""
+
+    families = ["model"]
+    representations = ["fried.ma", "fried.mb"]
+
```

Comment:
Introducing new loader plugin is mainly for representation separation and different icon. However this makes the scene manager tool fails to switch fried model to any other regular representation.